doc/tools/docgram/common.edit_mlg

```diff
@@ -12,6 +12,9 @@
 
 DOC_GRAMMAR
 
+(* defined in ml *)
+generic_tactic: [ ]
+
```

Comment:
Doc readers will want to know exactly what a `generic_tactic` is compared to other tactic nonterminals.  That should include a landing point for the hyperlink on the token.  Look at how it's handled for l1_tactic:

in ltac.rst
```- :production:`l1_tactic`\s are:```

in common.edit_mlg (suppresses doc_gram error message):
```
NOTINRSTS: [
   :
| l1_tactic
```